### DISCUSSION TITLE
tutorial-extras/key

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#6] [comment]
# tutorial-extras/key

简介 本文将介绍使用 Air001 开发板驱动按键使LED灯闪烁。 LED灯需要杜邦线连接Air001开发板。 硬件准备 LED灯，Air001开发板（自带灯）。 LED灯Air001 :----::----: GNDGND VCC3.3V INPA04| 软件部分 ::: 代码块 1. 定义Button状态变量，定义LED状态并赋值为LOW。 2. ...

https://arduino.luatos.com/tutorial-extras/key.html

**Comment by TZ-MSL** [comment]
iic和Spi也加点例程就好了
